cmake/packaging/linux.cmake

```diff
@@ -22,7 +22,7 @@ set(CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA "${SUNSHINE_SOURCE_ASSETS_DIR}/linux/misc
 set(CPACK_RPM_POST_INSTALL_SCRIPT_FILE "${SUNSHINE_SOURCE_ASSETS_DIR}/linux/misc/postinst")
 
 # Dependencies
-set(CPACK_DEB_COMPONENT_INSTALL ON)
+#set(CPACK_DEB_COMPONENT_INSTALL ON)
```

Comment:
Removing this puts all components into a single deb.

Need to ensure that gtest and gmock are not included (likely applies to all CPACK package types)